met-web/src/components/survey/submit/EngagementLink.tsx

```diff
@@ -14,7 +14,7 @@ export const EngagementLink = () => {
         return null;
     }
     return (
-        <MuiLink component={Link} to={`/engagement/view/${savedEngagement.id}`}>
+        <MuiLink component={Link} to={`/engagements/${savedEngagement.id}/view`}>
```

Comment:
Good catch, I'm starting to hate this routes everywhere, I'm thinking we could have some sort of constants for the routes, no change on this PR though